BUILD.md

```diff
@@ -738,40 +741,37 @@ To address this problem, run:
 
     make install
 
-This step "cleans up" the `RPATH` to remove any external references and
-performs other bundle fix-ups. After running `make install`, re-run the
-`otool` command again and note:
+This step copies the bundled applications to the location specified by
+CMAKE_INSTALL_PREFIX and "cleans up" the `RPATH` to remove any external
+references and performs other bundle fix-ups. After running `make install`,
+run the `otool` command again from the `build/install` directory and note:
 
 - `LC_LOAD_DYLIB` is now `@executable_path/../MacOS/libvulkan.1.dylib`
 - `LC_RPATH` is no longer present
 
 The "bundle fix-up" operation also puts a copy of the Vulkan loader into the
 bundle, making the bundle completely self-contained and self-referencing.
 
-Note that the "install" target has a very different meaning compared to the
-Linux "make install" target. The Linux "install" copies the targets to system
-directories. In MacOS, "install" means fixing up application bundles. In both
-cases, the "install" target operations clean up the `RPATH`.
-
 ##### The Non-bundled vulkaninfo Application
 
 There is also a non-bundled version of the `vulkaninfo` application that you
 can run from the command line:
 
     vulkaninfo/vulkaninfo
 
-If you run this before you run "make install", vulkaninfo's RPATH is already
+If you run this from the build directory, vulkaninfo's RPATH is already
 set to point to the Vulkan loader in the build tree, so it has no trouble
 finding it. But the loader will not find the MoltenVK driver and you'll see a
 message about an incompatible driver. To remedy this:
 
-    VK_ICD_FILENAMES=<path-to>/MoltenVK/Package/Latest/MoltenVK/macOS/MoltenVK_icd.json demos/vulkaninfo
+    VK_ICD_FILENAMES=<path-to>/MoltenVK/Package/Latest/MoltenVK/macOS/MoltenVK_icd.json vulkaninfo/vulkaninfo
 
```

Comment:
I know we discussed this offline, but if the old docs here are correct, there was no RPATH set in the vulkaninfo executable in the install location and we were specifying that setting DYLD_LIBRARY_PATH was necessary to find the loader.  I think we need to keep this behavior.  The `../lib` relationship is much too fragile.  And we shouldn't be "making it easy" to run things out of the repo's install tree(s).  The point of the build tree is to make things easier to run (hence, the use of RPATH there), but the point of the install tree is to have binaries free of RPATH entries that may have absolute or relative filesystem references.

Reply:
I removed INSTALL_RPATH from the vulkaninfo target and changed BUILD.md back to the original wording about RPATH being removed after "make install"